src/app/App.jsx

```diff
@@ -34,6 +35,7 @@ function App({ authState }) {
             <Route path="/recap/:locationId" component={withTracker(Recap)} />
             <Route path="/location/:locationId" component={withTracker(LocationInfo)} />
             <Route path="/questions/:id/:locationId" component={withTracker(LocationForm)} />
+            <Route path="/services" component={withTracker(ServiceContainer)} />
```

Comment:
@nlaz  You will probably want to include the locationId in the /services route so that on page refresh you can continue working on the services flow for that location.

Reply:
Yeah, it would probably make sense to actually structure the front-end URLs kinda like REST, e.g.:
```
<Route path="/location/:locationId" component={withTracker(LocationInfo)} />
<Route path="/location/:locationId/recap" component={withTracker(Recap)} />
<Route path="/location/:locationId/questions/:id" component={withTracker(LocationForm)} />
<Route path="/location/:locationId/services" component={withTracker(ServiceCategoriesContainer)} />
<Route path="/location/:locationId/services/recap" component={withTracker(ServiceForm)} />
<Route path="/location/:locationId/services/:serviceId" component={withTracker(ServiceForm)} />
```
etc.